backend/src/routes/aggregators/applications.ts

```diff
@@ -0,0 +1,319 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { getKubeResources } from '../events'
+import { AggregatedCacheType } from '../aggregator'
+import { getOCPApps } from './applicationsOCP'
+import { getArgoApps } from './applicationsArgo'
+import { IResource } from '../../resources/resource'
+import { FilterSelections, FilterCounts, ITransformedResource } from '../../lib/pagination'
+
+interface IArgoApplication extends IResource {
+  cluster?: string
+  spec: {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [x: string]: any
+    destination: {
+      name?: string
+      namespace: string
+      server?: string
+    }
+  }
+  status?: {
+    cluster?: string
+    decisions?: [{ clusterName: string }]
+  }
+}
+interface IOCPApplication extends IResource {
+  label?: string
+  status?: {
+    cluster?: string
+  }
+}
+interface IDecision extends IResource {
+  status?: {
+    decisions?: [{ clusterName: string }]
+  }
+}
+interface ISubscription extends IResource {
+  spec?: {
+    placement?: {
+      placementRef?: {
+        name: string
+      }
+    }
+  }
+  status?: {
+    decisions?: [{ clusterName: string }]
+  }
+}
+
+export async function aggregateApplications(aggregatedCache: AggregatedCacheType, key: string): Promise<void> {
+  // ACM Apps
+  const acmApps = getKubeResources('Application', 'app.k8s.io/v1beta1')
+
+  // Argo Apps
+  const { argoApps, argoAppSet } = await getArgoApps()
+
+  // Argo AppSets
+  const argoAppSets = getKubeResources('ApplicationSet', 'argoproj.io/v1alpha1')
+
+  // OCP Apps
+  const ocpApps = await getOCPApps(argoAppSet)
+
+  const filterCounts: FilterCounts = {}
+  const subscriptions = getKubeResources('Subscription', 'apps.open-cluster-management.io/v1')
+  const placementDecisions = getKubeResources('PlacementDecision', 'cluster.open-cluster-management.io/v1beta1')
```

Comment:
The original frontend code looks at Channels, PlacementRules, and Placements as well... I wonder if we are missing something here.